articles/openstack/ostack-how-basic-heat-template.md

```diff
@@ -0,0 +1,164 @@
+---
+title: How to create and deploy a basic heat template  | UKCloud Ltd
+description: Instructions on how to deploy a basic heat template
+services: openstack
+author: Bryce Nicholls
+toc_rootlink: How to
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create and deploy a basic heat template 
+toc_fullpath: How To/ostack-how-static-ip.md
```

Comment:
Incorrect file name here.
Also, should change the file name to ostack-how-create-heat-template.md (to better match the standards of how to file names including a verb after the how)